docs/index.txt

```diff
@@ -100,20 +101,18 @@ Running down the left-hand side, from top to bottom, the tabs are:
 
 - `Overview screen`_
 
-- :ref:`SQL console <console>`
-
-- :ref:`Tables browser <tables>`
+- :ref:`SQL console <sql-console>`
 
-- :ref:`Views browser <views>`
```

Comment:
The views tab has been added with the last big commit.

Reply:
ah damn! ok. I'll re-download. ty

Reply:
:-)

Reply:
this is too much work for now, I'll make a note and add this page once the changes land on crate/crate master